lambda/translattribute.ml

```diff
@@ -207,7 +245,7 @@ let add_inline_attribute expr loc attributes =
             (Warnings.Duplicated_attribute "inline")
       end;
       let attr = { attr with inline } in
-      check_local_inline loc attr;
+      check_poll_inline loc attr;
```

Comment:
@sadiqj why was the `check_local_inline` call removed here, when it is kept below? I wonder if it's a small bug or an intentional change.

Reply:
Good spot, I think this is a bug.